### PR TITLE
Remove application.yml from admin

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- This will enable other non-admin contexts (especially the module contexts in case of local module deployer) not to use the
application.yml properties

### CODE REVIEW

File: spring-cloud-data-admin/src/main/resources/bootstrap.yml [hunk 1]
```diff
@@ -1,9 +1,33 @@
+server:
```
Comment (ericbottard) [inline spring-cloud-data-admin/src/main/resources/bootstrap.yml]: I don't think `bootstrap.yml` is meant to contain props that are actually applicative properties. Its aim is to _bootstrap_, _i.e._ to tell where very early things, like the config server, are.

**Comment by ericbottard** [comment]
> This will enable other non-admin contexts (especially the module contexts in case of local module deployer) not to use the application.yml properties

not quite sure I understand what you mean, but in anycase, the module context should not "see" the admin context anyway (ClassLoader wise)

**Comment by markfisher** [comment]
Is it worth considering a `@PropertySource` for these properties instead? Agreed, the *real* solution will avoid any classpath visibility, but this is causing a critical bug at the moment.

**Comment by ericbottard** [comment]
merged as 73ef50f

**Pull request closed without merging**